docs/source/miniconda.rst

```diff
@@ -1,42 +1,58 @@
- |      | //repo.anaconda.co | //repo.anaconda.co | //repo.anaconda.co |
- |      | m/miniconda/Minico | m/miniconda/Minico | m/miniconda/Minico |
- |      | nda2-latest-Window | nda2-latest-MacOSX | nda2-latest-Linux- |
- |      | s-x86_64.exe>`__   | -x86_64.sh>`__     | x86_64.sh>`__      |
- |      |                    |                    |                    |
- |      | `32-bit (exe       | `64-bit (.pkg      | `32-bit (bash      |
- |      | installer) <https: | installer) <https: | installer) <https: |
- |      | //repo.anaconda.co | //repo.anaconda.co | //repo.anaconda.co |
- |      | m/miniconda/Minico | m/miniconda/Minico | m/miniconda/Minico |
- |      | nda2-latest-Window | nda2-latest-MacOSX | nda2-latest-Linux- |
- |      | s-x86.exe>`__      | -x86_64.pkg>`__    | x86.sh>`__         |
- +------+--------------------+--------------------+--------------------+
-
- `Installation
- instructions <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`__
-
- Other resources:
+Miniconda is a free minimal installer for conda. It is a small, bootstrap
+version of Anaconda that includes only conda, Python, the packages they depend
+on, and a small number of other useful packages, including pip, zlib and a
+few others. Use the ``conda install command`` to install 720+ additional conda
+packages from the Anaconda repository.
+
+`See if Miniconda is right for you <https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html#anaconda-or-miniconda>`_.
+
+Windows installers
+==================
+
+.. csv-table:: Windows
+   :header: Python version,Name,Size,SHA256 hash
+   :widths: 5, 10, 5, 80
+
+   Python 3.7,`Miniconda3 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe>`_,76.2 MiB,``a3a8921c2dec37f4ef37b9fa7b337dba237ccacec56bed3d8b8c300ed852c84f``
+   ,`Miniconda3 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86.exe>`_,70.7 MiB,``789a0cafbc4c43fb53facced1a32203865bc1600e5baf70e97e0ce3d64aebd4b``
+   Python 2.7,`Miniconda2 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda2-latest-Windows-x86_64.exe>`_,75.2 MiB,``9cf92cb336fd29c4fabbf22523d71a52623bf5ed7895d6cd079d569af5e4b7cd``
+   ,`Miniconda2 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda2-latest-Windows-x86.exe>`_,69.5 MiB,``a90d5b689f8a57c0da85ad77d3efa683a23da9ddb19429587635d222d5d1005c``
+
+
+MacOSX installers
+=================
+
+.. csv-table:: MacOSX
+   :header: Python version,Name,Size,SHA256 hash
+   :widths: 5, 10, 5, 80
+
+   Python 3.7,`Miniconda3 MacOSX 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh>`_,54.6 MiB,``c8b31ea37b0b6a3e2fb19990ef895ab5cf1c095f8e9138defac95ee88e70920d``
+   ,`Miniconda3 MacOSX 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg>`_,84.4 MiB,``e51804f0a55b1aac2200bbe21f06fe519536071ec14c8cb6d29f1ae7ec5dbfaf``
```

Comment:
This size doesn't match what I'm seeing, even though the hash matches, so I'm guessing this was human error (either mine or yours 😄).  What I see is 68.0MiB or 71.3MB. (71284529 bytes)